_pages/antipatterns.md

```diff
@@ -0,0 +1,112 @@
+---
+layout: default
+title: Pair Programming Antipatterns
+permalink: /antipatterns
+meta: What you shouldn't do when pair programming.
+---
+
+{% include article-top.html %}
+
+It's possible to pair well simply by avoiding pairing poorly. 
+
+Stay away from these common mistakes and you'll up your chances of success.
+
+## For navigators:
+
+### Leaping on errors too quickly
```

Comment:
Out of curiosity; why did you choose to name all headings after what **not** to do? I think I've heard doing the opposite yields better results because of psychology. :)

Reply:
I like that it's a little weird, honestly :)

Reply:
Fair enough. :) Great post BTW. Was greatly appreciated by me and my colleagues. :)